CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Features
+
+- feat: Add ANR tracking (#1861) 
```

Comment:
`h`: We (@bruno-garcia, @bruno-garcia and I) decided to align with the Apple ecosystem to name this App Hangs instead of ANR.